mkdocs.yml

```diff
@@ -70,9 +71,21 @@ plugins:
       nav_file: SUMMARY.md
   - section-index
   - mkdocstrings
+  - mknotebooks
 
 nav:
   - Getting started: index.md
   - Concepts: concepts.md
-  - Guides: guides.md
-  - Code Reference: reference/
+  - Learn:
+    - learn/index.md
+    - Tutorials:
+      - learn/tutorials/index.md
+    - User Guides:
+      - learn/user-guides/index.md
```

Comment:
To me the difference between these sub-sections is not super clear, maybe we could unify those under `Tutorials`, not sure about the differences though